### ISSUE TITLE
Support `-V` as an alias for `--version`

### ISSUE THREAD

**Issue by Ran4** [PyCQA/pyflakes#505]
Many tools in the python ecosystem supports showing version number using the `-V` argument.

Example, showing the version number of python, pip or mypy:

```
python -V
pip -V
mypy -V
```

I suggest adding `-V` as an alias to `--version`.

**Comment by asottile** [comment]
would you like to send a patch?

**Comment by asottile** [comment]
looks like #499 does this if you'd like to review that

**Comment by sim-d** [comment]
This would be trivial to support, e.g. from #499 by having:
```python
parser.add_argument('-V', '--version', action='version', version=_get_version())
```
instead of:
```python
parser.add_argument('--version', action='version', version=_get_version())
```

I just pushed that change to my fork, as a new commit on the same branch I started for #499: https://github.com/sim-d/pyflakes/tree/deprecate_optparse

Though I'm not sure what the recommend way of going forward would be. (New pull request? Update #499?) Or even if the maintainers would want to make this change.
